.github/workflows/test-template.yml

```diff
@@ -462,13 +462,13 @@ jobs:
     timeout-minutes: 35
     runs-on: ${{ matrix.os }}
 
-    if: ${{ contains(inputs.jobsToRun, '-all-') || contains(inputs.jobsToRun, '-client-') }}
+    if: ${{ contains(inputs.jobsToRun, '-all-') || contains(inputs.jobsToRun, '-client-e2e-') }}
 
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        node: [16]
+        node: [20]
```

Comment:
Why the switch now? (we still support Node.js 16)

I also want to switch, though for most cases I think it would be better to actually switch to 18 instead 🤔

Reply:
Right, then 20 is better with a comment